tests/policy_engine/test_policy.py

```diff
@@ -33,9 +33,14 @@ def test_dump_prelude(database_setup) -> None:  # type: ignore # pylint: disable
 def test_eval_policy(database_setup) -> None:  # type: ignore # pylint: disable=unused-argument,redefined-outer-name
     """Test loading the policy from file."""
     res = run_souffle(os.path.join(POLICY_FILE, DATABASE_FILE), POLICY_FILE)
-    res.pop("repo_satisfies_policy")
-    res.pop("repo_violates_policy")
+    # res.pop("component_satisfies_policy")
```

Comment:
Why are we commenting out these two lines?

Reply:
Because we should check their values. However, I should have removed the comments altogether, see commit [9522278](https://github.com/oracle/macaron/pull/305/commits/9522278852151c9565c4def4c25d69b73d602575)